app/models/organization.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
I suppose you add
```
Style/FrozenStringLiteralComment:
  Enabled: false
```
to `.rubocop.yml` for disabling `frozen_string_literal `.

Reply:
You can do it by yourself in your own issue)

Reply:
It's a bad idea, to open a new issue for each style thing)
In my opinion, we can close this conversation and decide, who & when have to add `rubocop cop`'s)

Reply:
Is it a good idea change one style thing in all project when you fixes another issue?